src/context/build-services.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable global-require */
+module.exports = (context) =>
+  context
+    .addInstance('logger', () => require('../services/logger'))
+    .addInstance('pathService', () => require('../services/path'))
+    .addInstance('errorHandler', () => require('../services/exposed/error-handler'))
+    .addInstance('ipWhitelist', () => require('../services/ip-whitelist'))
+    .addInstance('forestServerRequester', () => require('../services/forest-server-requester'))
+    .addInstance('schemasGenerator', () => require('../generators/schemas'))
+    .addInstance('baseFilterParser', () => require('../services/base-filters-parser'))
+    .addInstance('permissionsFormatter', () => require('../services/permissions-formatter'))
+    .addUsingClass('projectDirectoryFinder', () => require('../services/project-directory-finder'))
+    .addUsingClass('configStore', () => require('../services/config-store'))
+    .addUsingClass('permissionsGetter', () => require('../services/permissions-getter'))
+    .addUsingClass('permissionsChecker', () => require('../services/permissions-checker'))
+    .addUsingClass('apimapFieldsFormater', () => require('../services/apimap-fields-formater'))
+    .addUsingClass('authorizationFinder', () => require('../services/authorization-finder'))
+    .addUsingClass('apimapSorter', () => require('../services/apimap-sorter'))
+    .addUsingClass('apimapSender', () => require('../services/apimap-sender'))
+    .addUsingClass('schemaFileUpdater', () => require('../services/schema-file-updater'))
+    .addUsingClass('scopeManager', () => require('../services/scope-manager'))
+    .addUsingClass('modelsManager', () => require('../services/models-manager'))
+    .addUsingClass('tokenService', () => require('../services/token'))
+    .addUsingClass('oidcConfigurationRetrieverService', () => require('../services/oidc-configuration-retriever'))
+    .addUsingClass('oidcClientManagerService', () => require('../services/oidc-client-manager'))
+    .addUsingClass('authenticationService', () => require('../services/authentication'))
+    .addUsingClass('smartActionFieldValidator', () => require('../services/smart-action-field-validator'))
+    .addUsingClass('smartActionHookService', () => require('../services/smart-action-hook-service'))
+    .addUsingClass('smartActionHookDeserializer', () => require('../deserializers/smart-action-hook'));
```

Comment:
[just-to-say]
many of these classes:
- contain uselessly 'inject' that gives test harder to write
- contain constant values at the root of the file
- inject 'env' instead of values directly
- missing assert the injections (with assertPresent)